src/wagtail_site_inheritance/wagtail_hooks.py

```diff
@@ -51,14 +51,14 @@ def update_or_create_copies(request, page):
     if not is_publishing:
         return
 
-    create_non_existing_pages(request, page, parent_page)
-    sync_existing_pages(request, page)
-
     # Mark edited page as modified
     models.PageInheritanceItem.objects.filter(
-        inherited_page=page, modified=False
+        page=page, modified=False
```

Comment:
Dit lijkt mij niet te kloppen

Reply:
Als in volgens mij klopte de query eerst wel toch??

Reply:
Dit heeft nooit gewerkt. Als ik 3S bewerk wil ik niet dat alle inherited pages naar "modified" worden gezet, maar alleen de 3S pagina. Dit heeft dan namelijk weer impact op de velden die geupdated mogen worden. Dat was dus ook de bug waar we tegenaan liepen. Omdat altijd alle pagina's modified waren, werden alle velden geupdated, niet alleen de velden die geupdated mochten worden.